ChessClient/BlazorWASM/Pages/ChessGame.razor

```diff
@@ -38,6 +52,59 @@
         if (ack.Status == (uint)AckTypes.Success) return;
         DialogService.Show<InformDialogComponent>("Failed to resign", new DialogOptions() { CloseOnEscapeKey = true });
     }
+
```

Comment:
So I see there is a lot of logic on the chess page now. Can we move to some object that will handle draws/resigns?